numpy/ma/core.py

```diff
@@ -5541,6 +5541,17 @@ def round(self, decimals=0, out=None):
         --------
         numpy.ndarray.round : corresponding function for ndarrays
         numpy.around : equivalent function
+
+        Examples
+        --------
+        >>> import numpy.ma as ma
+        >>> x = np.ma.array([1.35, 2.5, 1.5, 1.75, 2.25, 2.75],
```

Comment:
You imported `ma` so you can do `x = ma.array(....` for consistency with existing examples in `ma/core.py`.